examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairOnNetworkLongCommand.java

```diff
@@ -15,5 +15,16 @@ public PairOnNetworkLongCommand(ChipDeviceController controller, CredentialsIssu
   }
 
   @Override
-  protected void runCommand() {}
+  protected void runCommand() {
+    currentCommissioner()
+        .pairDeviceWithAddress(
+            getNodeId(),
+            getRemoteAddr().getHostAddress(),
+            5540,
```

Comment:
Could we use a pre-defined const instead of hard coded value

Reply:
done, thanks